src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure {
```

Comment:
```suggestion
public abstract class Figure implements FigurePainter, AreaCalculator {
```
тоді наслідники автоматично буду імплементувати ці інтерфейси